libs/estdlib/src/erlang.erl

```diff
@@ -786,8 +788,31 @@ spawn(_Name) ->
 %% @end
 %%-----------------------------------------------------------------------------
 -spec spawn(Module :: module(), Function :: atom(), Args :: [any()]) -> pid().
-spawn(_Module, _Function, _Args) ->
-    erlang:nif_error(undefined).
+spawn(Module, Function, Args) ->
+    erlang:spawn_opt(Module, Function, Args, []).
```

Comment:
So the only issue I see with this is that some cases may now require estdlib in the “path”.  For some testing (not mine, but maybe @bettio ?) that might be an issue.  I personally don’t have an issue with this, however.  I think anyone who uses `erlang:` modules should be including `estdlib` (or `stdlib` and `kernel`, if we end up splitting them up), just for future compatibility.  But I understand this could be controversial.  Anyone else have any strong opinions?

Reply:
I do have the strong opinion that we shouldn't call a nif without the module itself. This is based on an assumption on how nifs handling work, we could require the module for the native functions to be loaded (so does BEAM).
It's ok with tests of nifs, though.

So we need estdlib, and I had to fix an example (emscripten) that didn't add estdlib and yet called erlang module functions.